__tests__/app.test.js

```diff
@@ -70,7 +70,60 @@ describe("api/reviews/:review_id", () => {
     });
   });
   describe("PATCH", () => {
-    test("should add a response body to the request object ", () => {});
+    test("Using the request body, should update the review object vote property ", () => {
+      const patchObj = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/reviews/2")
+        .send(patchObj)
+        .expect(200)
+        .then(({ body }) => {
+          const { updatedReview } = body;
+          const testReview = {
+            review_id: 2,
+            title: "Jenga",
+            designer: "Leslie Scott",
+            owner: "philippaclaire9",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            review_body: "Fiddly fun for all the family",
+            category: "dexterity",
+            created_at: expect.any(String),
+            votes: 6,
+          };
+          expect(updatedReview).toEqual(testReview);
+          expect(updatedReview.votes).toBe(6);
```

Comment:
94 - is this needed?  Is this being checked for with line 93?

Reply:
Haz told me to have that 2nd expect statement in